commands/build.go

```diff
@@ -75,6 +77,7 @@ var buildCmd = &cobra.Command{
 				 [--parallel PARALLEL_DEPTH]
 				 [--build-arg KEY=VALUE]
 				 [--build-option VALUE]
+				 [--copy-extra PATH]
```

Comment:
What is the priority of `--copy-extra` when a stack file is given?

Reply:
It is merged with the value from the stack file